builds/e2e/e2e.yaml

```diff
@@ -48,6 +48,91 @@ jobs:
       identityServiceArtifactName: packages_debian-11-slim_arm32v7
       identityServicePackageFilter: aziot-identity-service_*_armhf.deb
       sas_uri: $[ dependencies.Token.outputs['generate.sas_uri'] ]
+      minimal: true
+
+    timeoutInMinutes: 120
+
+    steps:
+    - template: templates/e2e-clean-directory.yaml
+    - template: templates/e2e-setup.yaml
+    - template: templates/e2e-clear-docker-cached-images.yaml
+    - template: templates/e2e-run.yaml
+      parameters:
+        sas_uri: $(sas_uri)
+
+################################################################################
+  - job: debian_12_amd64
+################################################################################
+    displayName: Debian 12 amd64
+    dependsOn: Token
+    condition: succeeded('Token')
+
+    pool:
+      name: $(pool.linux.name)
+      demands:
+      - ImageOverride -equals agent-aziotedge-debian-12-msmoby
+
+    variables:
+      os: linux
+      arch: amd64
+      artifactName: iotedged-debian12-amd64
+      identityServiceArtifactName: packages_debian-12-slim_amd64
+      identityServicePackageFilter: aziot-identity-service_*_amd64.deb
+      sas_uri: $[ dependencies.Token.outputs['generate.sas_uri'] ]
+
+    timeoutInMinutes: 90
+
+    steps:
+    - template: templates/e2e-setup.yaml
+    - template: templates/e2e-run.yaml
+      parameters:
+        sas_uri: $(sas_uri)
+
+################################################################################
+  - job: debian_12_arm64v8
+################################################################################
+    displayName: Debian 12 arm64v8
+    dependsOn: Token
+    condition: succeeded('Token')
+
+    pool:
+      name: $(pool.custom.name)
+      demands: debian12-arm64-e2e-tests
+
+    variables:
+      os: linux
+      arch: arm64v8
+      artifactName: iotedged-debian12-aarch64
+      identityServiceArtifactName: packages_debian-12-slim_aarch64
+      identityServicePackageFilter: aziot-identity-service_*_arm64.deb
+      sas_uri: $[ dependencies.Token.outputs['generate.sas_uri'] ]
+
+    timeoutInMinutes: 90
+
+    steps:
```

Comment:
I think we need 
``` 
    - template: templates/e2e-clean-directory.yaml
    - template: templates/e2e-clear-docker-cached-images.yaml
```
on ARM as well since they are self host, right?

Reply:
Yes thanks for catching that!

Reply:
Fixed. I'm re-running the end-to-end pipeline...

Reply:
Confirmed the Debian 12 arm64 job passes.